script.js

```diff
@@ -0,0 +1,210 @@
+  return `<li>${el}</li>`;
+ });
+ buildElement(projectEl.img_source, projectEl.title, projectEl.subtitile, midList[0], midList[1], 
+  projectEl.description, listStack, projectEl.id, projectEl.sbutton);
+})
+
+
+/************************** MODAL SECTION **************************************/
+var ebModal = document.getElementById('mainModalArea');
+
+// Get the button that opens the modal
+//var ebBtn = document.querySelector(".show_modal");
+document.querySelectorAll('.show_modal').forEach((row) => row.addEventListener('click', () => displayModal(row.getAttribute('data-id'))));
+
+
+const displayModal = (j) => { 
+  //console.log(project_array[j])
+ 
+  let count = j-1;
+  console.log(project_array[count])
+
+  let listStackpop = project_array[count].list_stack.map(el => {
+    return `<li>${el}</li>`;
+   });
+
+
+  document.querySelector('#popup-section').innerHTML = `
+  <div class="pop-container">
+  <h2>${project_array[count].title}</h2>
+  <span class="subtitle">${project_array[count].subtitile} </span>
+  <span class="mid-list">
+    <span class="dot"></span>${project_array[count].mid_list[0]}
+    <span class="dot"></span>${project_array[count].mid_list[1]}
+  </span>
+  <div class="flex-items-pop"><img src="${project_array[count].img_source}" alt="Portfolio" /></div>
+  <div class="flex-pop-foot">
+    <div class="description">${project_array[count].description}</div>
+    <ul class="list-stack"> 
+     ${listStackpop.join(' ')}
+     <div>
+     <br/><br/>
+    <a href='https://ogiogiovictor.github.io/microverse_porfolio_project/' class="btn-see-project">See Live <i class="fa fa-external-link" aria-hidden="true"></i></a>
+    <a href='https://github.com/ogiogiovictor' class="btn-see-project">See Source <i class="fa fa-github-square" aria-hidden="true"></i></a>
+   </div>
+    </ul>
+  </div>
+  
+  
+  </div>
+  `
+
+  ebModal.style.display = "block";
+}
+// Get the <span> element that closes the modal
+var ebSpan = document.getElementsByClassName("close_modal")[0];
+
+// When the user clicks the button, open the modal 
+// ebBtn.onclick = function() {
+//     ebModal.style.display = "block";
+// }
+
+// When the user clicks on <span> (x), close the modal
+ebSpan.onclick = function() {
+    ebModal.style.display = "none";
+}
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function(event) {
+    if (event.target == ebModal) {
+        ebModal.style.display = "none";
+    }
+}
+
+
+/********************* FORM VALIDATION ****************************** */
+const form = document.getElementById('contactForm');
+const email = form.elements['email'];
+const fullname = document.querySelector('#fullname');
+const message = document.querySelector('#message');
+//const submit = document.querySelector('#form-btn');
+//Check the email if is in lowercase
+
+form.addEventListener('submit', function (e) {
+  // prevent the form from submitting
+  e.preventDefault();
```

Comment:
- [ ] Hi please as part of the requirement, when all the fields are filled properly the form is supposed to send the message. Since you've prevented default action, kindly make sure to send the form when every thing is ok